unassigned_issues/unassigned_issues_scraper.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from django.http import HttpResponse
+from gci.config import get_api_key
+from IGitt.GitHub import GitHubToken
+from IGitt.GitHub.GitHubRepository import GitHubRepository
+
+from community.git import get_org_name
+
+
+def run(mr_requests):
+    issues_number_list = []
+    for pr in mr_requests:
+        if pr.state._name_ == 'OPEN':
+            for commit in pr.commits:
+                mystatus = commit.combined_status._name_
+                break
+            if mystatus == 'SUCCESS' or mystatus == 'PENDING':
+                for issue in pr.closes_issues:
+                    if str(pr.number) != issue.number:
+                        if str(issue.state) == 'open':
+                            if len(issue.assignees) is 0:
```

Comment:
>List of all the issues on organization main repository on which someone opened a PR (with passed ``CI``) without getting assigned to it

According to your statement, you're getting the list of issues on which `a person opened a PR but that person is not assigned to it`. But according to your code here, you are getting the list of issues on which `a person opened a PR but no one is assigned to the issue. `  Right? 
What if a person opened a PR and he is not assigned to the issue but someone else is? You should get that issue into the list, but you're not getting it.

Reply:
It was done intentionally, If someone is already assigned on it then maintainers not have to bother , they will only have to bother one issue is open and it have no assignee.

`` I will change my commit message clear understanding`

Reply:
changed.

Reply:
I agree with @sks444, you should check if pr.author is assigned to that issue or not, number of assignees doesn't matter. Maintainers will add him if needed else will close that pr.

Reply:
I will include that. :+1: